BlueprintUI/Sources/Internal/HintingSizeCache.swift

```diff
@@ -38,6 +95,47 @@ final class HintingSizeCache {
 
         values[key] = size
 
+        // This option adds (or "hints") extra cache keys along the boundaries of the range
+        // between the constraint and the measured size.
+        if options.hintRangeBoundaries {
```

Comment:
Going to need to read this a couple more times, but I think I follow!

Reply:
I'm going to be the guy that says a diagram here or elsewhere and linked here would be hugely helpful. I had the same take as Kyle's.